3rdparty/find_dependencies.cmake

```diff
@@ -997,7 +997,7 @@ if(NOT USE_SYSTEM_FMT)
     # We set the FMT_HEADER_ONLY macro, so no need to actually compile the source
```

Comment:
This comment can be removed now, since we compile `fmt` now.

Reply:
Done,